### PR TITLE
extend docs notes

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?
Reaction to #821. Adding notes how to build docs locally

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: docs/source/introduction_guide.rst [hunk 1]
```diff
@@ -201,8 +202,10 @@ the LightningModule
     class CoolMNIST(pl.LightningModule):
 
       def train_dataloader(self):
-        transform=transforms.Compose([transforms.ToTensor(), transforms.Normalize((0.1307,), (0.3081,))])
-        mnist_train = MNIST(os.getcwd(), train=True, download=False, transform=transform)
+        transform=transforms.Compose([transforms.ToTensor(),
+                                      transforms.Normalize((0.1307,), (0.3081,))])
+        mnist_train = MNIST(os.getcwd(), train=True, download=False,
+                            transform=transform)
         return DataLoader(mnist_train, batch_size=64)
```
Comment (ethanwharris) [inline docs/source/introduction_guide.rst]: Shouldn't the dataloaders have `shuffle=True` for training?

Comment (Borda) [reply docs/source/introduction_guide.rst]: lets move it to another PR :]

**Comment by williamFalcon** [comment]
@ethanwharris @Borda 
fixed the introduction guide docs.

Also fixed that assignment in prepare_data wasn't working. now it is

**Comment by williamFalcon** [comment]
@Borda any final doc changes? we're ready to go

**Comment by Borda** [comment]
@williamFalcon is it a typo `LitMNIST` ? >> `LightMNIST`

**Comment by williamFalcon** [comment]
not a typo
https://www.urbandictionary.com/define.php?term=lit
